data/json/monsters/zed_misc.json

```diff
@@ -479,6 +479,66 @@
     ],
     "armor": { "bash": 8, "cut": 5, "bullet": 4, "electric": 2 }
   },
+  {
+    "id": "mon_zombie_grue",
+    "type": "MONSTER",
+    "name": { "str": "zombie prowler" },
+    "description": "Enveloped in darkness is the outline of what used to be a person, quick and spindly - and nigh invisible in the shadows.  Even with the unnatural camouflage hiding its features you can make out large teeth and even larger claws.",
+    "default_faction": "zombie",
+    "looks_like": "mon_zombie_brute_ninja",
+    "bodytype": "human",
+    "species": [ "ZOMBIE", "HUMAN" ],
+    "diff": 2,
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 110,
+    "speed": 115,
+    "material": "flesh",
+    "symbol": "Z",
+    "color": "dark_gray_red",
+    "scents_tracked": [ "sc_human", "sc_fetid" ],
+    "aggression": 100,
+    "melee_skill": 5,
+    "melee_dice": 4,
+    "melee_dice_sides": 4,
+    "melee_damage": [ { "damage_type": "cut", "amount": 5 } ],
```

Comment:
I recommend against going with bump damage for the main output - adding a beefy scratch_humanoid would work better and prevents damage spikes (since mobs get one special per move, whereas they would get to put out 21 dmg on top of whatever special they rolled with this).